web/src/app/browser/src/keymanEngine.ts

```diff
@@ -38,6 +38,11 @@ export default class KeymanEngine extends KeymanEngineBase<BrowserConfiguration,
   hotkeyManager: HotkeyManager = new HotkeyManager();
   private readonly beepHandler: BeepHandler;
 
+  /**
+   * Provides a quick link to the base help page for Keyman keyboards.
+   *
+   * See https://help.keyman.com/developer/engine/web/17.0/reference/core/helpURL
```

Comment:
```suggestion
   * See https://help.keyman.com/developer/engine/web/current-version/reference/core/helpURL
```

The `current-version` URL will map consistently to the latest release version, and avoid us having to update these comments with each release. Yes, there may be a window of time where it doesn't find the file for new docs for an alpha release, but I think we can live with that.